test/unit/globecontrol.js

```diff
@@ -152,13 +152,13 @@ describe('GlobeControls', function () {
     });
 
     it('dolly', function () {
-        controls.dollyIn(10);
+        controls.dolly(10);
         controls.state = controls.states.ORBIT;
         controls.update();
-        controls.dollyOut(10);
+        controls.dolly(0.1);
         controls.state = controls.states.NONE;
-        controls.dollyOut();
-        controls.dollyIn();
```

Comment:
I removed these two lines which were just here to test the condition `if (dollyScale === undefined)` in `dollyIn` and `dollyOut` methods.